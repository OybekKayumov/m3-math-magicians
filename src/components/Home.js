import './pages.css';

const Home = () => {
  const para = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo labore et dolore magna aliqua.';

  return (
    <section className="home-container">
      <h1>Welcome To Our page!</h1>
      <p>{para}</p>
    </section>
  );
};

export default Home;
