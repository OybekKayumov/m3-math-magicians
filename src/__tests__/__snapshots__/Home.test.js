import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../components/Home';

it('render correctly', () => {
  const elements = renderer.create(<Home />).toJSON();

  expect(elements).toMatchSnapshot();
});
