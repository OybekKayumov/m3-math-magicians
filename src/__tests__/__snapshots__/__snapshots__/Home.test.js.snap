// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<section
  className="container-home"
>
  <h1>
    Welcome To Our Page!
  </h1>
  <p>
    Description: Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to Make simple calculations, Read a random math-related quote. By building this application, you will learn how to use React. How to build the "Math magicians" website.   The core functionality is a calculator. At the very beginning your website will contain only the simple UI presented below, but it will be able to perform all math operations.
  </p>
</section>
`;
